### ISSUE TITLE
Would love Visual Studio Support

### ISSUE THREAD

**Issue by disturbing** [MiguelYax/ApiDocSnippets#4]
It says VSCode only on the marketplace.  Any way we can get it on visual studio as well?

Just fell in love with API Doc <3.

Also - any way to ensure error checking? @apiPadm would error (or at least suggest it to be @apiParam)

**Comment by MiguelYax** [comment]
hello @Disturbing 

This is a interesting suggestion.  But this is not possible because vscode snippets definition is not same visual studio snippet definition.  At the moment this snippets only support vscode.

If you interesting on build a visual studio snippet? 

I can help you on building this...

**Comment by disturbing** [comment]
Yes, i am interested, but only know csharp. Maybe we can discuss how to
collaborate?
